src/Kernel/client/kernel.ts

```diff
@@ -21,9 +21,56 @@ class Kernel {
             this.requestEcho();
             this.requestClientInfo();
             this.setupMeasurementHistogramDataListener();
+            this.setupMeasurementHistogramDebugger();
         });
     }
 
+    setupMeasurementHistogramDebugger() {
+        IPython.notebook.kernel.register_iopub_handler(
+            "iqsharp_debug_opstart",
+            message => {
+                console.log("iqsharp_debug_opstart message received", message);
+
+                //check to see if there's already a button
```

Comment:
Is this a TODO?

Reply:
I commented for notetaking reasons, but I think the comment can be removed!